### PR TITLE
Refactor pa global state

### PR DESCRIPTION
**Pull request by jepidoptera**
Still a work in progress. Specifically, graphing utility needs to be revamped.

### CODE REVIEW

File: hydradx/model/amm/global_state.py [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+
+    def price(self, asset):
+        return self.external_market[asset] if asset in self.external_market else 0
+
+    def copy(self):
+        self_copy = copy.deepcopy(self)
+        return self_copy
+
+
+def fluctuate_prices(state: GlobalState, percent: float, bias: float):
```
Comment (poliwop) [inline hydradx/model/amm/global_state.py:71]: I like that you've refactored this market evolution step out. There's very standard ways to evolve market prices under common assumptions that asset values follow brownian motion. I think this is a decent place to start for reading:
https://en.wikipedia.org/wiki/Brownian_model_of_financial_markets
I have some books on this too, it's all a little math-intensive but the actual implementations wind up being pretty intuitive. So having the ability to plug in different functions for different kinds of asset value evolution is great.

**Review by poliwop — COMMENTED** [review]
Looked pretty quickly at the architecture, looks good to me.

File: hydradx/model/run.py [hunk 2]
```diff
@@ -1,36 +1,23 @@
-from cadCAD import configs
```
Comment (poliwop) [inline hydradx/model/run.py:1]: I know I don't usually suggest we do this, but can we keep the cadCAD integration code here but commented out?

Comment (poliwop) [reply hydradx/model/run.py:1]: or somehow in the repo, but not used... maybe in a different run.py file? not sure what makes it easiest to switch back and forth

**Review by poliwop — COMMENTED** [review]
Quickly reviewed, I saw no issue with any of the changes

**Review by jepidoptera — COMMENTED** [review]
Looks good to me!

**Pull request merged**